source/particles/particle_handler.cc

```diff
@@ -455,26 +475,50 @@ namespace Particles
 
 
 
+  template <int dim, int spacedim>
+  void
```

Comment:
would it make sense to let this function return a particle_iterator? The one other function called `insert_particle` does that at the moment.